index.js

```diff
@@ -166,66 +168,130 @@ function provider ({ Biome, blocks, blocksByStateId, toolMultipliers, shapes, ma
     return heldItemType && this.harvestTools && this.harvestTools[heldItemType]
   }
 
-  function effectLevel (effect, effects) {
-    const e = effects[effectsByName[effect].id]
+  // Determine data required to actually compute dig times that is version-dependent
+  let blockDigTimeData
+
+  // 1.17+: effect names have been fixed to actually match their registry names
+  if (protocolVersion >= 755) {
```

Comment:
I disagree with the choice to do this instead of `mcData.isNewerOrEqualTo()`, it makes version checking a lot less readable thus harder to understand for new contributors. I know that above you say what the version should be, but that's not necessary. as another option, possibly a `features.json` could be added to mcdata or to this package alone like in mineflayer to allow for describing the version difference as well